docs/introduction/install.rst

```diff
@@ -33,18 +33,18 @@ Installing on Windows OS:
   
      git clone git@github.com:HorizenOfficial/Sidechains-SDK.git
     
-  5. As IDE, please install and use IntelliJ IDEA Community Edition (link) In the IDE, please also install the Intellij Scala plugin: in the Settings->Plugins tab, select it from the marketplace: 
+  5. As IDE, please install and use IntelliJ IDEA Community Edition (link). In the IDE, please also install the Intellij Scala plugin: in the Settings->Plugins tab, select it from the marketplace: 
   
   .. image:: /images/intellij.png
    :alt: IntelliJ
   
-  6. In the IDE, you can now  go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file, the Maven’s Project Object Model XML file that contains all the project configuration details should be automatically imported by the IDE. Otherwise, you can just open it.
-  7. Keep reading this tutorial, and start playing with the code. You will find some sidechain examples in the “examples/simpleapp” directory, that you can customize, start from there! When you are ready to run your standalone sidechain, you can install Maven (`link <https://maven.apache.org/install.html>`_).
-  8. To produce your specific sidechain jar files, you can change directory to the repository root and run the “mvn package” command.   
+  6. In the IDE, you can now go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file, the Maven Project's Object Model XML file that contains all the project configuration details, should be automatically imported by the IDE. Otherwise, you can just open it.
```

Comment:
@zennermac 

  6. In the IDE, you can now go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file, the Maven Project's Object Model XML file that contains all the project configuration details should be automatically imported by the IDE. Otherwise, you can just open it.

The section "The pom.xml file, the Maven Project's Object Model XML file" doesn't make sense. Can you message me about this Mac?